plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TrinoCatalogFactory.java

```diff
@@ -49,7 +51,8 @@ public TrinoCatalogFactory(
             TypeManager typeManager,
             IcebergTableOperationsProvider tableOperationsProvider,
             NodeVersion nodeVersion,
-            IcebergSecurityConfig securityConfig)
+            IcebergSecurityConfig securityConfig,
+            HiveConfig hiveConfig)
```

Comment:
I brought in `HiveConfig` here because it's used specifically to produce a <code>Trino*Hive*Catalog</code>, but maybe I should copy the config property into `IcebergConfig` instead?